app/views/pages/partials/_footer.html.erb

```diff
@@ -0,0 +1,27 @@
+<footer>
+  <div class="top-wave"></div>
+  <div class="footer-container">
+    <%= vite_image_tag "images/illustrations/1010.png", class: "plant-bottom" %>
+    <div class="content-logo">
+      <%= vite_image_tag "images/logos/logo_text_black_clipped.png", class: "black-logo" %>
+    </div>
+    <div class="content-links">
+      <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Sessions</a></li>
+        <li><a href="#">Videos</a></li>
+        <li><a class="btn-contact" href="#">Contact me</a></li>
+      </ul>
+    </div>
```

Comment:
A list of links should be a `<nav>`, you also don't need to nest that inside a div, just give the content-links class to the nav directly

Reply:
done